libraries/cli/include/cli/default_config.hpp

```diff
@@ -202,8 +203,11 @@ const char *default_json = R"foo({
           "disable_nonce_check": true
         },
         "genesis_balances": {
-          "0274cfffea9fa850e54c93a23042f12a87358a82": "0x141e8d17",
-          "111f91441efc8c6c0edf6534970cc887e2fabaa8": "0x24048ce3d"
+          "0274cfffea9fa850e54c93a23042f12a87358a82": "0x141e8d17000000000000000000",
+          "111f91441efc8c6c0edf6534970cc887e2fabaa8": "0x24048ce3d000000000000000000"
+        },
+        "hardforks": {
+          "fix_genesis_hardfork_block_num": "0x12c"
```

Comment:
so dont we want to start using tip_<NUM> as `Taraxa Improvement Proposal` naming for hardforks ?

Reply:
Did we agreed on how we do it? We could crate TIP describing this hardfork and name it `tip1_block_num`. But we should have that TIP first

Reply:
maybe not even _num this is how ETH is doing it 
```    "homesteadBlock": 0,
    "eip150Block": 0,
    "eip155Block": 0,
    "eip158Block": 0,
    "byzantiumBlock": 0,
    "constantinopleBlock": 0,
    "petersburgBlock": 0,
```

Reply:
I remeber @justinsnapp agreed on calling it tip, not sure about the others though ?

Reply:
Agreed there that to make TIP name we should have some TIP description in a proper place for it. As we didn't have one it makes sense to call it with a "speaking name"